interfaces/input_parameters/ring.py

```diff
@@ -35,72 +38,23 @@ class Ring:
-        Momentum compaction factor of zeroth order :math:`\alpha_{0,k,i}` [1];
-        can be input as single float or as a program of (n_turns + 1) turns
-        (should be of the same size as synchronous_data).
-        In case of higher order momentum compaction, check the
-        documentation for the inputs: alpha_order, alpha_1, alpha_2
-    synchronous_data : float (opt: float array/matrix [n_sections, n_turns+1])
-        Design synchronous particle momentum (default) [eV], kinetic or
-        total energy [eV] or bending field [T] on the design orbit.
-        Input for each RF section :math:`p_{s,k,n}`.
-        Can be input as a single constant float, or as a
-        program of (n_turns + 1) turns. In case of several sections without
-        acceleration, input: [[momentum_section_1], [momentum_section_2],
-        etc.]. In case of several sections with acceleration, input:
-        [momentum_program_section_1, momentum_program_section_2, etc.]. Can
-        be input also as a tuple of time and momentum, see also
-        'cycle_time' and 'PreprocessRamp'
     Particle : class
         A Particle-based class defining the primary, synchronous particle (mass
         and charge) that is reference for the momentum/energy in the ring.
-    n_turns : int
-        Optional: Number of turns :math:`n` [1] to be simulated.
-        If a synchronous_data program is passed as a tuple (see below),
-        the number of turns will be overwritten depending on the length in time
-        of the program
-    synchronous_data_type : str
-        Optional: Choice of 'synchronous_data' type; can be 'momentum'
-        (default), 'total energy', 'kinetic energy' or 'bending field'
-        (requires bending_radius to be defined)
-    bending_radius : float
-        Optional: Radius [m] of the bending magnets,
-        required if 'bending field' is set for the synchronous_data_type
-    n_sections : int
-        Optional: number of ring sections/segments; default is 1
-    alpha_1 : float (opt: float array/matrix [n_sections, n_turns+1])
-        Momentum compaction factor of first order
-        :math:`\alpha_{1,k,i}` [1]; can be input as single float or as a
-        program of (n_turns + 1) turns (should be of the same size as
-        synchronous_data and alpha_0).
-    alpha_2 : float (opt: float array/matrix [n_sections, n_turns+1])
-        Optional : Momentum compaction factor of second order
-        :math:`\alpha_{2,k,i}` [1]; can be input as single float or as a
-        program of (n_turns + 1) turns (should be of the same size as
-        synchronous_data and alpha_0).
-    RingOptions : class
-        Optional : A RingOptions-based class with default options to check the
-        input and initialize the momentum program for the simulation.
-        This object defines the interpolation scheme, plotting options, etc.
-        The options for this object can be adjusted and passed to the Ring
-        object.
+    Section_list : list
```

Comment:
Work specifying what it's a list of?  And making it iterable rather than list?

Reply:
yes definitely, I'll keep the explicit mention to list as the beginner user may not know what's an iterable in python

Reply:
And renamed Section_list to RingSection_list for naming consistency